src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-OLTP-SYSBENCH.json

```diff
@@ -8,135 +8,169 @@
     },
     "Parameters": {
         "DatabaseName": "sbtest",
-        "DatabaseScenario": "Balanced",
-        "Duration": "00:05:00"
+        "NumTables": "10",
```

Comment:
Couple of things:
1) Naming -> TableCount

2) The user will need to have the ability to manage the number of records in each of the tables as well. I see the conundrum here between supporting 3 scenarios: 1) balanced, 2) in-memory or 3) user-specified # of tables and records. So, let's just make a decision now. For OUR profile, we will support the DatabaseScenario (Balanced or InMemory) out of the box. However, we will still support the ability of the user to supply a TableCount and RecordCount parameter in the SysbenchConfiguration for populating the database. We just won't expose it here in our profile. The ABC team can create their own profiles if they want to have a database of some specific size. This profile is NOT for their purposes.

Let's sync on this when you have a sec so we are on both in agreement with our "pattern"